app/classifier/index.cjsx

```diff
@@ -250,17 +250,11 @@ Classifier = React.createClass
     </div>
 
   renderExpertOptions: ->
-    if @props.project?
-      getUserRoles = @props.project.get 'project_roles'
+    if @props.project? && @props.user?
+      getUserRoles = @props.project.get('project_roles', user_id: @props.user.id)
```

Comment:
Bad design on my part. I expected most resources to be fetched as links of other resources, so I conflated making specific queries with wanting fresh data. Clearly not the case.

Ideally when we encounter problems like this we'll use it as an opportunity to stop using a `PromiseRenderer`.